internal/server/server.go

```diff
@@ -203,6 +203,8 @@ func New(options Options) (*Server, error) {
 
 	configureEmail(options)
 
+	server.SetupBackgroundJobs()
```

Comment:
Background jobs should be started by `Run`, not by `New`. This is really important for testing.

Should be easy to move this to `Run`.